android-test/build.gradle

```diff
@@ -1,6 +1,12 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 
+repositories {
+  maven {
+    url 'https://repo.spring.io/plugins-release'
```

Comment:
Fixes running inside Android Studio

https://stackoverflow.com/questions/49573157/android-studio-3-1-could-not-find-org-jetbrains-trove4jtrove4j20160824

Reply:
You should use `jcenter()` for this. And ideally restrict its usage only to that artifact with:
```groovy
    jcenter {
      // Required for a dependency of Android lint.
      content {
        includeGroup 'org.jetbrains.trove4j'
      }
    }
```

Reply:
Any thoughts on jcenter... :)